src/erl/clj_analyzer.erl

```diff
@@ -854,8 +854,10 @@ parse_binding({Pattern, Init}, Env0) ->
                    false -> 'let'
                  end,
 
-  {PatternExpr, Env1} = clj_env:pop_expr(parse_pattern(Pattern, Env0)),
-  {InitExpr, Env2}    = clj_env:pop_expr(analyze_form(Init, Env1)),
+  {InitExpr, Env1}    = clj_env:pop_expr(analyze_form(Init, Env0)),
+  Env2                = clj_env:push(#{is_binding_context => true}, Env1),
```

Comment:
Only indicate a binding context for the pattern in the binding, not the initial value.